src/openms/include/OpenMS/ANALYSIS/TOPDOWN/FLASHDeconvHelperStructs.h

```diff
@@ -208,7 +208,7 @@ namespace OpenMS
         //       @param mz mz
```

Comment:
minor: indentation and comments seem a bit off here. This could result in parameters not showing up as expected in the doxygen documentation

Reply:
because this is a public member you should also add some information what is returned (e.g., what is the effect of the parameter "positive" e.g, I asked myself "why not a getLogAbsMz without this parameter"

Reply:
Done.